README.md

```diff
@@ -12,14 +12,28 @@ on-chain fee.
 
 More on how this works [here](https://github.com/hsjoberg/blixt-wallet/issues/242).
 
+## Dunder's API
+
+The API endpoints supplied by dunder LSP are:
+
+- /estimateFee - returns estimated fee (sats) and fee rate (sats per byte)
+
+- /getInfo - returns info from lnd, see https://github.com/hsjoberg/dunder-lsp/blob/master/proto/rpc.proto#L1596 and look for GetInfoReponse to understand the returned data. Exact line numbers may vary.
+
+- /ondemand-channel
+    - /check-status - checks validity and sats claimable.
+    - /claim - rarely neded. Used to check status of a request in case /register does not succeed to give full feedback.
```

Comment:
Slightly wrong.

```diff
-     - /claim - rarely neded. Used to check status of a request in case /register does not succeed to give full feedback.
+     - /claim - rarely needed. Used to claim funds available on the Dunder server in case /register fails to open a channel.